ansible/roles/build-ossec-deb-pkg/tasks/main.yml

```diff
@@ -21,7 +21,7 @@
   with_items: build_ossec_deb_pkg_dependencies
 
 - name: extract OSSEC archive to /tmp
-  shell: tar -xvf {{ build_path }}/{{ download_name }} -C /tmp
+  shell: tar -xvf {{ build_path }}/ossec-hids-{{ download_name }} -C /tmp
```

Comment:
I moved `ossec-hids` out of the `download_name` because the `download_url` + `download_name` otherwise would not match the new url.